include/neml2/tensors/LabeledMatrix.h

```diff
@@ -38,6 +38,10 @@ class LabeledMatrix : public LabeledTensor<1, 2>
   /// Conversion from LabeledTensor
   LabeledMatrix(const LabeledTensor<1, 2> & other);
 
+  static LabeledMatrix zeros(TorchShapeRef batch_size,
+                             const std::vector<const LabeledAxis *> & axes,
+                             const torch::TensorOptions & options = default_tensor_options);
```

Comment:
The original zero constructor led to ambiguity. I am making this a static method now,